tensorrt_llm/executor/worker.py

```
@@ -512,6 +512,9 @@ def _deduce_max_tokens(request: GenerationRequest,
                 else:
                     # use max_tokens if can't deduce default_max_tokens
                     return max_tokens
+            assert (
```

Ruff (0.12.2)

Use of `assert` detected

(S101)